src/vm/runners/builtin_runner/ec_op.rs

```diff
@@ -188,8 +189,13 @@ impl BuiltinRunner for EcOpBuiltinRunner {
         self
     }
 
-    fn get_memory_accesses(&self, _runner: &CairoRunner) -> std::collections::HashSet<Relocatable> {
-        todo!()
+    fn get_memory_accesses(&self, vm: &VirtualMachine) -> Result<Vec<Relocatable>, MemoryError> {
+        let segment_size = vm
+            .segments
+            .get_segment_size(self.base as _)
```

Comment:
Use try_into() here instead of casting

Reply:
Should I avoid isize to usize casts even when it's impossible for a value to be negative?

Reply:
The type of base is isize, so we can't asume it won't be negative, we should instead create an issue to change it into usize